docs/NEMU + XS-GEM5 使用流程.md

```
@@ -0,0 +1,698 @@
+
+​	本教程的主要目的是帮助初学者完成nemu + xs-gem5 的全流程使用，具体来说，主要是详细说明了以下三个方面该如何执行：
+
+1. ​	如何产生基于 opensbi 的自定义负载
+
+2. ​	如何在 NEMU 上执行上述产生的负载并生成对应切片
+
+3. ​	如何能够在 XS-GEM5 上成功运行上述切片并得到相关输出结果
+
+   本教程主要依赖以下教程与自身实验的总结：
+
+   https://docs.xiangshan.cc/zh-cn/latest/workloads/opensbi-kernel-for-xs/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)